lms/templates/courseware/accordion.html

```diff
@@ -40,9 +40,12 @@
                 ## a status icon
                 <%
                     if section.get('due') is None:
-                        data_string = ''
+                        data_string = section['format']
                     else:
-                        data_string = _('due {date}')
+                        if 'proctoring' in section:
```

Comment:
It was how it was written in a previous version -- 
```
% if 'proctoring' in section:
<span class="subtitle-name">${due_date}</span>
 % else:
<span class="subtitle-name">${section['format']} ${due_date}</span>
...
```
I'm not sure beyond that, I tried to leave the original conditions in place as much as possible. If there's no good reason, happy to take out.